### DISCUSSION TITLE
HPC Hackathon?

### DISCUSSION THREAD

**Discussion by hestes835** [hpc-social/hpc-social.github.io#58] [Ideas]
Hello @alansill; as you said, moving this conversation to the GitHub discussions will be better. 

In my time running the local ACM chapter at my university, our yearly hackathon has always been our biggest event and always gets us more members. It would be some extra work, but an HPC-oriented hackathon could help spread the word and be a fun activity. What do you think? 

We would need something that sets us apart due to the online setting. I think "HPC in The City" did not happen this year. I am unsure of other HPC-oriented hackathons, so that may be enough to get a following.

**Comment by sramkrishna** [comment]
What kind of ideas do you do for hackathons?
